src/routes/userRoute.js

```diff
@@ -0,0 +1,31 @@
+const express = require('express');
```

Comment:
As I understand, these routes are not connected in the main file - index.js

Reply:
changed and connected